docs/book/src/topics/rosa/support.md

```diff
@@ -0,0 +1,17 @@
+# Create issue for ROSA
+
+When creating issue for ROSA-HCP cluster, include the logs for the capa-controller-manager and capi-controller-manager deployment pods. The logs can be saved to text file using the commands below. As well as, include the CRs that is used to create the ROSA ControlPlane, MachinePools and Cluster.
+
+```shell
+$ kubectl get pod -n capa-system 
+NAME                                      READY   STATUS    RESTARTS   AGE
+capa-controller-manager-77f5b946b-sddcg   1/1     Running   1          3d3h
+
+$ kubectl logs -n capa-system capa-controller-manager-77f5b946b-sddcg &>> capa-controller-manager-logs.txt
```

Comment:
why `&>>` ? this will append to end of file, so if the file had any previous content/logs it will be confusing to read.
`>` should be good.

Reply:
okay

Reply:
The logs are only STDOUT, we don't need to redirect STDERR with `&>` 
if  `kubectl logs` throws the error, the user should see it and fix it before they can save the logs to a file. 
please use only `>`